src/components/Policy/PolicyTable.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+import { Table, TableHeader, TableBody, IRow, IActions, ICell, sortable, OnSelect } from '@patternfly/react-table';
+import {
+    Bullseye,
+    EmptyState,
+    EmptyStateBody,
+    EmptyStateIcon,
+    EmptyStateVariant,
+    Title
+} from '@patternfly/react-core';
+import { Spinner } from '@patternfly/react-core/dist/js/experimental';
+import { Policy } from '../../types/Policy';
+import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
+
+interface PolicyTableProps {
+    policies?: Policy[];
+    actions?: IActions;
+    loading?: boolean;
+    hasError?: boolean;
+    onSelect?: OnSelect;
+}
+
+const loading = (): IRow[] => [{
+    heightAuto: true,
+    cells: [
+        {
+            props: { colSpan: 8 },
+            title: (
+                <Bullseye>
+                    <EmptyState variant={ EmptyStateVariant.small }>
+                        <EmptyStateIcon variant="container" component={ Spinner }/>
+                        <Title size="lg">Loading</Title>
+                    </EmptyState>
+                </Bullseye>
+            )
+        }
+    ]
+}];
+
+const error = (): IRow[] => [{
+    heightAuto: true,
+    showSelect: false,
+    cells: [
+        {
+            props: { colSpan: 8 },
+            title: (
+                <Bullseye>
+                    <EmptyState variant={ EmptyStateVariant.small }>
+                        <EmptyStateIcon icon={ ExclamationCircleIcon }/>
+                        <Title size="lg">Unable to connect</Title>
```

Comment:
This error also comes when the backend has no data and returns a 404.

Reply:
Ok, I'll try to do a more fine error handling here.